app/javascript/react/components/ReviewsContainer.js

```diff
@@ -2,9 +2,23 @@ import React from "react";
 import ReviewTile from "./ReviewTile";
 
 const ReviewsContainer = (props) => {
+
   if (props.reviews.length > 0) {
     const reviewArray = props.reviews.map((review) => {
-      return <ReviewTile key={review.id} rating={review.rating} comment={review.comment}/>;
+
+      const handleClick = () => {
+        props.deleteReview(review.id)
+      }
+
+      return <ReviewTile 
+        key={review.id} 
+        id={review.id} 
+        rating={review.rating} 
+        comment={review.comment} 
+        handleClick={handleClick} 
+        currentUser={props.currentUser}
+      />;
```

Comment:
I get the intention behind this indentation, but we do unfortunately want towards:
- wrapping what is in your return in `()`
- ensuring consistent indentation for the React component invocation

due to the number of props you are aiming to pass 
https://github.com/airbnb/javascript/tree/master/react#alignment